packages/language-service/test/lib/diagnostics.spec.ts

```diff
@@ -158,7 +158,10 @@ describe('diagnostics', () => {
                 filePath
             );
 
-            expect(diagnostics).to.eql([]);
+            expect(diagnostics).to.have.length(1);
+            expect(diagnostics[0]).to.deep.include({
+                message: processorWarnings.DEPRECATED_ST_GLOBAL_CUSTOM_PROPERTY(),
+            });
```

Comment:
I can see that the deprecation diagnostic is now emited, but the test doesn't make sense even before this change. It would be better to remove it imo.